ledger/ledger-api-common/src/main/scala/com/digitalasset/platform/server/api/validation/ErrorFactories.scala

```diff
@@ -52,8 +52,13 @@ trait ErrorFactories {
   def resourceExhausted(description: String): StatusRuntimeException =
     grpcError(Status.RESOURCE_EXHAUSTED.withDescription(description))
 
+  def participantPrunedDataAccessed(message: String): StatusRuntimeException =
+    grpcError(Status.NOT_FOUND.withDescription(message))
+
   def grpcError(status: Status) = new ApiException(status)
 
+  def fromGrpcStatus(status: Status): StatusRuntimeException = grpcError(status)
```

Comment:
This doesn't look like it's adding anything. Why not calling the method above directly?